modules/face/src/getlandmarks.cpp

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
Can you replace all giant headers by

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

This is in principle more than enough and removes unnecessary content